src/lib/context-menu.ts

```diff
@@ -72,17 +72,32 @@ export class ContextMenu {
   }
 
   public showAddMenu = (target: Element, after: boolean) => {
+    const { nodes } = this.view.state.schema
+    const { childNodes } = this.view.dom
+    const isBibliographySection = [...childNodes].some((section: ChildNode) => {
+      if (section.nodeType === Node.ELEMENT_NODE) {
+        const sectionEl = section as Element
+        return (
+          section?.childNodes[0]?.pmViewDesc?.node === this.node &&
+          sectionEl.classList.contains('bibliography')
+        )
+      }
+      return false
+    })
+    // we don`t want to add section after 'REFERENCES'
+    after =
```

Comment:
It is considered a bad practice to mutate function arguments. Can we use a local variable instead?